tests/Schema/BlueprintTest.php

```
@@ -234,12 +234,29 @@ public function test_create_indexes(): void
             "create unique index `{$indexPrefix}_name_unique` on `{$tableName}` (`name`)",
             "create index `{$indexPrefix}_createdat_index` on `{$tableName}` (`createdAt`)",
+            "create index `{$indexPrefix}_name_createdat_index` on `{$tableName}` (`name`, `createdAt`)",
+        ], $statements);
+    }
+
+    public function test_create_indexes_with_order(): void
+    {
+        $conn = $this->getDefaultConnection();
+        $tableName = $this->generateTableName();
+        $indexPrefix = Str::snake($tableName);
+        $blueprint = new Blueprint($conn, $tableName, function (Blueprint $table) {
```

PHPStan (2.2.8)

[error] Parameter `#3` $callback of class Colopl\Spanner\Schema\Blueprint constructor expects (Closure(Illuminate\Database\Schema\Blueprint): void)|null, Closure(Colopl\Spanner\Schema\Blueprint): void given.


(argument.type)